codewindtechpreview.md

```diff
@@ -58,7 +58,7 @@ Projects must be located in `<user>/microclimate-workspace`.
 #### Installing the Eclipse plug-in
 1. Download and install the latest Eclipse IDE for Java EE Developers or use an existing installation. The earliest supported version of the Eclipse IDE is Version 4.8 (Photon).
 2. Obtain the plug-in in one of the following ways: 
-    - Open the [Eclipse IDE](https://marketplace.eclipse.org/content/codewind) and go to **Help**>**Eclipse Marketplace**. Then, search for `Codewind`. Click **Install** and finish the wizard and accept licenses as prompted.
+   - [![Drag this install button to your running Eclipse workspace. Note: You need to have the Eclipse Marketplace Client.](https://marketplace.eclipse.org/sites/all/themes/solstice/public/images/marketplace/btn-install.png)](http://marketplace.eclipse.org/marketplace-client-intro?mpc_install=4638524 "Drag to your running Eclipse* workspace. *Requires Eclipse Marketplace Client") Drag this install button to your running Eclipse workspace. Note: You need to have the Eclipse Marketplace Client.
```

Comment:
The new drag-to-install instructions should replace the second list item, not the first

Reply:
@jopit So, it should be the second bullet point and not the first?

Reply:
Oh wait...I understand now. I have fixed it in another PR.